pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,48 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infoBox = container.find('.info');
+    var content = infoBox.html();
+    var textArea = container.find('.read-write-info textarea').val($.trim(content));
+    container.find('.edit-info').hide();
+    container.find('.read-only-info').hide();
+    container.find('.read-write-info').toggleClass('hidden');
```

Comment:
You can replace `.hide()` and `.show()` with `.toggle()` to get the same effect. Then you have the pattern of these three lines repeating 3 times. You can turn it into a function, e.g.:

```js
function toggleWidgets() {
  container
    .find('.edit-info').toggle().end()
    .find('.read-only-info').toggle().end()
    .find('.read-write-info').toggleClass('hidden');
}
```